src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -29,11 +29,14 @@ internal class HttpInMetricsListener : ListenerHandler
 
         private Histogram<double> httpServerDuration;
 
+        private Counter<long> httpServerCompletedRequests;
+
         public HttpInMetricsListener(string name, Meter meter)
             : base(name)
         {
             this.meter = meter;
             this.httpServerDuration = meter.CreateHistogram<double>("http.server.duration", "ms", "measures the duration of the inbound HTTP request");
+            this.httpServerCompletedRequests = meter.CreateCounter<long>("http.server.completed_requests", "requests", "counts the HTTP requests that are completed");
```

Comment:
The instrumentation libraries are based on the SemanticConventions (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md).
Adding new metric or changing the meaning, would have to go through updating the semantic conventions first. Its okay to have prototypes and gather feedback, but before merging, we need to have it part of spec.

All semantic conventions are currently experimental, and are subject to breaking changes. (that is the main reason the instrumentation libraries are still non-stable).

Reply:
yeah, @reyang was pointing me in that direction, too. After talking through it and thinking about this more, I realized early this morning that this PR just won't do. There needs to be a larger conversation on these metrics.

My plan is to get rid of this PR, then open up a new issue on our project to discuss this more, moving the conversation out of this soon-to-be abandoned PR. Once I think we're fleshed this out more on what makes sense, I was planning to open a PR to the spec project.

This definitely turned out more complex than I originally thought...maybe I should stick to updates to the test environments 😆